packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -310,7 +289,7 @@ describeNoCompat("Container", (getTestObjectProvider) => {
 		deltaConnection.removeAllListeners();
 	});
 
-	it("Delta manager receives readonly event when calling container.forceReadonly()", async () => {
+	it("Delta manager receives readonly event when calling container.forceReadonly?.()", async () => {
```

Comment:
Will changing the test name affect test stability stuff or no?

Reply:
```suggestion
	it("Delta manager receives readonly event when calling container.forceReadonly()", async () => {
```

Reply:
it shouldn't affect anything, but i will fix it